### PR TITLE
Repo Clean Up and Volume Row Improvements

### PR DESCRIPTION
**Pull request by bnussman-akamai**
## Description 📝

- Removes some `__data__` files that were no longer used. We prefer using factories for generating test data. (See `packages/manager/src/factories/*`)
- Removes old unused Volumes code
- Improves flawed volume testing for volumes table row

## How to test 🧪

- Test Volumes table rows at `http://localhost:3000/volumes`
- Test volumes table rows on a Linode that has atleast one volume attached at `http://localhost:3000/linodes/:id/storage`

### CODE REVIEW

File: packages/manager/src/features/Volumes/VolumeTableRow.tsx [hunk 1]
```diff
@@ -142,7 +142,7 @@ export const VolumeTableRow: React.FC<CombinedProps> = (props) => {
       ) : null}
       {isVolumesLanding && (
         <TableCell data-qa-volume-cell-attachment={linode_label}>
-          {linodeId ? (
+          {linodeId !== null ? (
```
Comment (bnussman-akamai) [inline packages/manager/src/features/Volumes/VolumeTableRow.tsx:145]: This is actually a bug fix here. If someone had a volume with `id` of `0`, the proper information would not show

**Review by mjac0bs — APPROVED** [review]
Thanks for the clean up to old `__data__ ` files! When testing a previous PR of my own with mock data, I remember accidentally making a change to `__data__/account.ts` rather than `account.ts` and not realizing what wasn't working at first. 🤦‍♀️ 

Just caught one typo correction in a comment. Volume table rows on Linode Details vs Volumes Landing look like they're showing the expected information. 🚢

File: packages/manager/src/features/Volumes/VolumeTableRow.tsx [hunk 2]
```diff
@@ -53,7 +55,7 @@ export const isVolumeUpdating = (e?: Event) => {
   );
 };
 
-export const volumeStatusIconMap: Record<ExtendedVolume['status'], Status> = {
```
[the remark this answers is not in the document]
Comment (bnussman-akamai) [reply packages/manager/src/features/Volumes/VolumeTableRow.tsx:56]: It was used to allow us to augment the Volumes data before we stored it in Redux, but now that Volumes is all React Query, we don't do any modification to the data at the time of storing it. We now just compute "extended" data at render time

**Review by jdamore-linode — APPROVED** [review]
Nice work Banks, thanks for taking this on!

**Pull request merged**